README.md

```diff
@@ -53,13 +62,18 @@ class Main extends Component {
 
 ### Question #4
 
-Products receives a property called `listing` containing an array of objects. Each object contains a key for `name` (a string) and `price` (a number). Edit the below code snippet to render a list of `Comment` components that take `name` and `price` as incoming properties.
+Products receives a property called `listing` containing an array of objects. Each object contains a key for `name` (a string) and `price` (a number). Edit the below code snippet to render a list of `Product` components that take `name` and `price` as incoming properties.
 
 ```js
 // assume Component and Product are imported above
 class Products extends Component {
   render () {
+      var listings = product.listing.map((list, index){
+          {list.price} {list.name}
```

Comment:
We were looking for use of the `Product` component:
```js
return( <Product name={list.name} price={list.price} key={index} /> )
```
we were also accepting the following:
```js
return( <div key={index}><p>{list.name}</p><p>{list.price}</p></div> )
```